src/blocks/author-profile/components/inspector/LayoutPanel.js

```
@@ -0,0 +1,309 @@
+					label={__('Border Width', 'author-profile-blocks')}
+					value={borderWidth}
+					onChange={(value) => setAttributes({ borderWidth: value })}
+					min={0}
+					max={10}
+					initialPosition={0}
+				/>
+
+				{borderWidth > 0 && (
+					<BaseControl
+						label={__('Border Color', 'author-profile-blocks')}
+					>
```

ESLint

[error] When using BaseControl component if a label property is passed an id property should also be passed.

(@wordpress/no-base-control-with-label-without-id)